crates/jstzd/tests/jstzd_test.rs

```diff
@@ -1,32 +1,57 @@
 mod utils;
+use std::io::Read;
 use std::path::PathBuf;
+use std::str::FromStr;
 
+use http::Uri;
 use jstzd::task::jstzd::{JstzdConfig, JstzdServer};
 use jstzd::task::utils::retry;
-use jstzd::{EXCHANGER_ADDRESS, JSTZ_NATIVE_BRIDGE_ADDRESS};
+use jstzd::{EXCHANGER_ADDRESS, JSTZ_NATIVE_BRIDGE_ADDRESS, JSTZ_ROLLUP_ADDRESS};
 use octez::r#async::baker::{BakerBinaryPath, OctezBakerConfigBuilder};
 use octez::r#async::client::{OctezClient, OctezClientConfigBuilder};
 use octez::r#async::endpoint::Endpoint;
 use octez::r#async::node_config::{OctezNodeConfigBuilder, OctezNodeRunOptionsBuilder};
 use octez::r#async::protocol::{
-    BootstrapAccount, BootstrapContract, ProtocolParameterBuilder,
+    BootstrapAccount, BootstrapContract, BootstrapSmartRollup, ProtocolParameterBuilder,
+    SmartRollupPvmKind,
 };
+use octez::r#async::rollup::{OctezRollupConfigBuilder, RollupDataDir};
 use octez::unused_port;
+use serde_json::from_str;
+use std::fs;
+use tempfile::NamedTempFile;
+use tezos_crypto_rs::hash::SmartRollupHash;
 
+include!(concat!(env!("OUT_DIR"), "/jstz_rollup_path.rs"));
```

Comment:
Its best to add this line in `src/lib.rs` then import it from there

Reply:
done